backend/src/apiserver/template/template.go

```diff
@@ -259,3 +344,86 @@ func toPipelineJobRuntimeConfig(apiRuntimeConfig *api.PipelineSpec_RuntimeConfig
 	runtimeConfig.GcsOutputDirectory = apiRuntimeConfig.GetPipelineRoot()
 	return runtimeConfig, nil
 }
+
+func modelToPipelineJobRuntimeConfig(modelRuntimeConfig *model.RuntimeConfig) (*pipelinespec.PipelineJob_RuntimeConfig, error) {
+	if modelRuntimeConfig == nil {
+		return nil, nil
+	}
+	parameters := new(map[string]*structpb.Value)
+	err := json.Unmarshal([]byte(modelRuntimeConfig.Parameters), parameters)
+	if err != nil {
+		return nil, err
+	}
+	runtimeConfig := &pipelinespec.PipelineJob_RuntimeConfig{}
+	runtimeConfig.ParameterValues = *parameters
+	runtimeConfig.GcsOutputDirectory = modelRuntimeConfig.PipelineRoot
+	return runtimeConfig, nil
+}
+
+func modelToCRDTrigger(modelJob model.Job) scheduledworkflow.Trigger {
+	crdCronSchedule := scheduledworkflow.CronSchedule{}
+	crdPeriodicSchedule := scheduledworkflow.PeriodicSchedule{}
+	if &modelJob.CronSchedule != nil {
```

Comment:
This doesn't seem right. The address of `CronSchedule` would never be `nil`.

Reply:
That's a really good point, and probably the reason why the presubmit test did not pass. I have updated the `modelToCRDTrigger` function.